### DISCUSSION TITLE
Lowering Streaming / httpd buffer?

### DISCUSSION THREAD

**Discussion by dr181818** [MusicPlayerDaemon/MPD#1632] [General]
As mentioned in the documentation, using the httpd Streaming option, has a long buffer time when feeds are switches. I've seen 30-50 seconds.   Is there any way to reduce the buffer size, to reduce the delay? 

Thanks

**Comment by sp1ff** [comment]
The `httpd` plugin works in terms of "pages", and they're hard-coded to be 32768 bytes in size (that will represent different lengths of time, depending on encoding). If you look at `HttpdOutput::HttpdOutput` there is no configuration value to affect this. I might take a run at making it configurable if you open an issue?

**Comment by dr181818** [comment]
Done. Thank you!
